File: Olympiad/COCI/official/2009/contest3_solutions/pet.cpp
#include <cstdio>
using namespace std;

int main() {
   int max_score = 0, winner = 0;
   for ( int i=1; i<=5; ++i ) {
      int sum = 0;
      for ( int j=0; j<4; ++j ) {
         int x;
         scanf("%d", &x);
         sum += x;
      }
      if ( sum > max_score ) {
         max_score = sum;
         winner = i;
      }
   }
   printf("%d %d\n", winner, max_score);

   return 0;
}

File: Olympiad/COCI/official/2009/contest3_solutions/pet.pas
program pet;

var
   max_score, winner, i, sum, j, x : integer;

begin
   max_score := 0;
   for i:=1 to 5 do begin
      sum := 0;
      for j:=1 to 4 do begin
         read(x);
         sum := sum+x;
      end;
      if sum > max_score then begin
         max_score := sum;
         winner := i;
      end;
   end;
   writeln(winner, ' ', max_score);
end.
